source/user_guide/case_auditing.rst

```diff
@@ -0,0 +1,138 @@
+.. currentmodule:: howso.engine
+
+
+Case Auditing
+=============
+.. topic:: What is covered in this user guide
+
+    In this guide, you will learn how to audit the cases in your Trainee, including determining which cases influence a specific prediction, as well as which cases influence the :class:`~Trainee` 's predictions as a whole.
+    
+
+Objectives: what you will take away
+-----------------------------------
+- **Definitions & Understanding** Difference between similar/influential cases vs case importance and robust vs non-robust :ref:`Case Contributions <user_guide/terminology:Contribution>` and :ref:`Case MDA <user_guide/terminology:MDA>`.
+- **How-To** audit specific cases as well as determine which cases impact Trainee predictions.
+- **API References** How to use :py:meth:`Trainee.react`.
+
+
+Prerequisites: before you begin
+-------------------------------
+- You have successfully :doc:`installed Howso Engine <../getting_started/installing>`
+- You have :doc:`loaded, configured, trained, and analyzed data <basic_workflow>`
+
+
+Notebook Recipe
+---------------
+The following recipe will supplement the content this guide will cover:
+
+- :download:`Interpretability <https://github.com/howsoai/howso-engine-recipes/blob/main/2-interpretability.ipynb>`
+
+
+Concepts & Terminology
+----------------------
+The main piece of terminology this guide introduces is the concept of case auditing. To understand this, we
+recommend being familiar with the following concepts:
+
+- :ref:`Trainee <user_guide/terminology:trainee>`
+- :ref:`React <user_guide/terminology:react>`
+- :ref:`Influential Cases <user_guide/terminology:Influential Cases>`
+- :ref:`Most Similar Cases <user_guide/terminology:Most Similar Cases>`
+- :ref:`Case MDA <user_guide/terminology:MDA>`
+- :ref:`Case Contributions <user_guide/terminology:Contribution>`
+
+
+How-To Guide
+------------
+
+
+Influential Cases and Most Similar Cases
+----------------------------------------
+
+Howso allows the user to audit individual cases to determine the closest cases that may influence a prediction. **Most Similar Cases** allows the user to return a designated number of most similar cases, 
+while the **Most Influential Cases** only returns cases that have an influence on the prediction. If the desired number of most similar cases is greater than the number of most influential cases, then the most influential 
+cases is a subset of the most similar cases. 
+
+The parameters to get the most similar cases or influential cases can be accessed in the ``details`` parameter as a key value pair shown below. ``num_most_similar_cases`` allows the user to specify exactly how many similar cases to return.
+
+.. code-block:: python
+
+    details = {
+        'influential_cases':True,
+        'most_similar_cases':True,
+        'num_most_similar_cases':20,
+    }
+
+    results = t.react(
```

Comment:
I'd rather this variable be named `reaction` rather than 'results', but probably not a super important issue